packages/@webex/plugin-meetings/test/unit/spec/rtcMetrics/index.ts

```diff
@@ -82,6 +87,22 @@ describe('RtcMetrics', () => {
     assert.callCount(webex.request, 1);
   });
 
+  it('should clear out metrics on failure', () => {
```

Comment:
we should have a test that the connectionId is the same between metrics.addMetrics when successful

Reply:
Updated